### ISSUE TITLE
Quantum Entangloporter Energy Transferring Issue

### ISSUE THREAD

**Issue by UnknownSerhan** [mekanism/Mekanism#8207]
### Issue description

When 2 pair of entangloporters that are transferring energy on 2 different frequencies touch each other, it seems like the entangloporter are acting like a single energy buffer or something like that. This happens as long as at least 2 entangloporters with different frequencies touch each other. Adding more than 2 entangloporters touching each other results in weird energy amounts being transferred (3 entangloporters touching makes the energy transfer rate 136.53 MFE/t). I do not know if this issue occurs only with energy transfer or others too.

### Do note that all sides of entangloporters are set to same side configuration. Input entangloporters' all sides are set to input, and output entangloporters' all sides are set to output. Also auto ejects of all entangloporters are enabled.

# Here is an example setup that works as expected:

Matrix 1 transmits energy to Matrix 2 with 2 pair of entangloporters that are on separate frequencies:
![image](https://github.com/user-attachments/assets/bb024d15-b494-4a5a-85cf-fec1c0c79a4e)

Matrix 1 entangloporter placement:
![2024-08-20_22 38 28](https://github.com/user-attachments/assets/4b1552af-ea80-4262-8599-b7e756c70ffb)

Matrix 1 GUI (the output is correct):
![image](https://github.com/user-attachments/assets/7f16cd2f-620b-4315-8236-075d00da0049)

Matrix 2 entangloporter placement:
![2024-08-20_22 39 34](https://github.com/user-attachments/assets/6d045c69-9e29-4e68-b159-26b719dfa3ed)

Matrix 2 GUI:
![image](https://github.com/user-attachments/assets/2a47376d-ab33-4ba5-8086-0f86f42663d1)

# And here is an example setup that does NOT work es expected:

Matrix 1 entangloporter placement (they are touching now):
![2024-08-20_22 46 28](https://github.com/user-attachments/assets/4a924ca0-5143-41a7-a7b4-42922f91ce15)

Matrix 1 GUI (see that the output has dropped to half):
![image](https://github.com/user-attachments/assets/8564959d-d542-410b-94d7-953ea8ef397c)

Matrix 2 entangloporter placement (they are touching now):
![2024-08-20_22 47 35](https://github.com/user-attachments/assets/1c3c99d7-1b16-4ed3-ab58-b1389be37a16)

Matrix 2 GUI (the input is now half because of Matrix 1 output rate):
![image](https://github.com/user-attachments/assets/b3b0787a-e260-48b6-9a88-f526537553e2)

_TLDR entangloporters touch each other seem like to share their buffer, which shouldn't happen in the first place._

### Steps to reproduce

The best way to reproduce and see this issue to build a setup like above and observe the energy transfer rates.

### Minecraft version

1.21 (Latest)

### NeoForge version

21.1.22

### Mekanism version

Current Dev Branch (Manually Compiled, specify git commit hash)

**Comment by UnknownSerhan** [comment]
Here is the world that you can easily observe the issue.

[Entangloporter Tests.zip](https://github.com/user-attachments/files/16734703/Entangloporter.Tests.zip)
